### ISSUE TITLE
Native Country Name?

### ISSUE THREAD

**Issue by lcarva** [rsertelon/iso-3166-country-codes-angular#17]
Would be great to have a way to display the country name in its native form not just in English. For example:

    'BR' -> 'Brasil',
    'IT' -> 'Italia'

**Comment by rsertelon** [comment]
This is not part of the ISO standard, I could to it however, but I'd need some help, for each existing country. Plus some countries have several official languages, which makes it a bit more complicated ;)

**Comment by lcarva** [comment]
I think this might contain all the needed data:https://github.com/hexorx/countries/blob/master/lib/cache/translations.yaml

**Comment by rsertelon** [comment]
Indeed. I don't have much time for now. But I'll try to work on it when possible.

**Comment by vovafeldman** [comment]
+1
